Docs/reference/content/reference/driver/crud/client_side_encryption.md

```diff
@@ -0,0 +1,206 @@
++++
+date = "2019-09-30T20:38:-04:00"
+title = "Client Side Encryption"
+[menu.main]
+  parent = "Reference Reading and Writing"
+  identifier = "Client-Side Field Level Encryption"
+  weight = 40
+  pre = "<i class='fa fa-lock'></i>"
++++
+
+# Client-Side Field Level Encryption
+
+New in MongoDB 4.2, client-side field level encryption allows administrators and
+developers to encrypt specific data fields in addition to other MongoDB
+encryption features.
+
+With client-side field level encryption, developers can encrypt fields
+client-side without any server-side configuration or directives. Client-side
+field level encryption supports workloads where applications must guarantee that
+unauthorized parties, including server administrators, cannot read the encrypted
+data.
+
+{{% note class="important" %}} 
+Support for client-side field level encryption is in beta, and the beta only
+supports Windows. Backwards-breaking changes may be made before the
+final release (which will support MacOS and Linux).
+{{% /note %}}
+
+## Installation
+
+The recommended way to get started using client-side field level encryption in
+your project is with a package manager such as NuGet. Client-side field level
+encryption requires one additional package, `MongoDB.Libmongocrypt`, in addition
+to the standard driver packages.
+
+See the [installation]({{< relref "getting_started\installation.md" >}}) for instructions on how to install the MongoDB driver.
+
+
+### libmongocrypt
+
+`libmongocrypt` is a required component for client-side field level encryption
+and is included in `MongoDB.Libmongocrypt` package.
```

Comment:
True... but is this TMI?

Reply:
Hmm a fair question. Including this line may help explain the appearance of the libmongocrypt binaries in application folders?

Reply:
Sure that could be useful information.

Reply:
Are we okay leaving this section in? :)

Reply:
Out of date, resolving.